test/conftest.py

```diff
@@ -753,12 +753,6 @@ def no_answer_reader(request):
         )
 
 
-@pytest.fixture
```

Comment:
I can confirm that `prediction` isn't used in any other tests except for the reader.py. However, it might be useful to have this fixture available to other nodes as well no?

Reply:
I don't think so. I think fixtures should stay close to the tests that use them, and this is such a small one I hardly see the point of having it at all. Let's keep it near the tests.